jhotdraw-samples/jhotdraw-samples-misc/src/main/java/org/jhotdraw/samples/pert/PertPanel.java

```diff
@@ -76,27 +76,20 @@ public PertPanel() {
     JMenu m = new JMenu(labels.getString("view.zoomFactor.text"));
     JRadioButtonMenuItem rbmi;
     ButtonGroup group = new ButtonGroup();
+
     m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 0.1, null)));
     group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 0.25, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 0.5, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 0.75, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 1.0, null)));
-    rbmi.setSelected(true);
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 1.25, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 1.5, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 2, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 3, null)));
-    group.add(rbmi);
-    m.add(rbmi = new JRadioButtonMenuItem(new ZoomAction(editor, 4, null)));
-    group.add(rbmi);
+    for (double zoomLevel = 0.25; zoomLevel <= 4.0; zoomLevel += 0.25) {
```

Comment:
Did you want to create the buttons for all the zoom levels up to 4? Just wanted to cross-check since the previous code did not have buttons for all the zoom levels in the range of 0.25 to 4.